src/test/java/bio/terra/CloudStorageEventHarnessTest.java

```diff
@@ -0,0 +1,24 @@
+package bio.terra;
+
+import static org.junit.Assert.fail;
+
+import bio.terra.cloudfunctions.common.GsonWrapper;
+import bio.terra.common.BaseTest;
+import com.google.events.cloud.storage.v1.StorageObjectData;
+import java.util.Map;
+import org.junit.Test;
+
+public class CloudStorageEventHarnessTest extends BaseTest {
+  @Test
+  public void acceptTest() {
+    try {
+      GCSEventHarnessImpl instance = new GCSEventHarnessImpl();
+      Map<?, ?> m = GsonWrapper.convertFromClass(MOCK_EVENT_GZIP, Map.class);
+      instance.accept(m, new CFContext());
+      StorageObjectData event = instance.getEvent(StorageObjectData.class);
+      assertStorageObjectEvent(event);
+    } catch (Exception e) {
+      fail(e.getMessage());
+    }
```

Comment:
I don't think the try/catch is necessary here - if the code throws an Exception, the test should fail, right?

Reply:
Now the `instance.accept(m, SUPPORTED_CLOUD_EVENT_CONTEXT);` statement might unexpectedly throw an unexpected event type exception. If that's the case, the `fail` statement in the catch-block will report the bug.

Reply:
`fail()` should be used when you expect an exception to be thrown and it's not.  The test below will fail if instance.accept() detects an unsupported event and throws an exception and is more idiomatic junit.
```
  @Test
  public void acceptSupportedCloudEventTest() throws Exception {
    GCSEventHarnessImpl instance = new GCSEventHarnessImpl();
    Map<?, ?> m = GsonWrapper.convertFromClass(MOCK_GCS_EVENT_GZIP, Map.class);
    instance.accept(m, GCS_CLOUD_EVENT_CONTEXT);
    StorageObjectData event = instance.getEvent(StorageObjectData.class);
    assertMockTGZStorageObjectData(event);
  }
```

Reply:
Removed `try-catch` and updated test to throw Exception when something goes wrong.